experimental/dds/tree/src/test/IdCompressor.tests.ts

```diff
@@ -595,6 +638,18 @@ describe('IdCompressor', () => {
 			expect(id).to.not.equal(normalized);
 		});
 
+		it('can normalize an eagerly generated final ID', () => {
+			const compressor = createCompressor(Client.Client1, 5);
+			compressor.generateCompressedId();
+			compressor.finalizeCreationRange(compressor.takeNextCreationRange());
+			const eagerFinalId = compressor.generateCompressedId();
+			expect(isFinalId(eagerFinalId)).to.be.true;
```

Comment:
This is in a lot of the new tests. I wonder if we should have a single glass box test that checks that the second (or more) generated IDs are eager-final, and then the rest of these tests don't need to check that. But maybe they should be self-describing in this way (i.e. they don't really need to exist if we don't have eagers). Not sure. Thoughts?

Reply:
Talked offline, agreed to do this in a follow-up PR.